src/utils/MainUtils.java

```diff
@@ -1,4 +1,12 @@
 package utils;
 
 public class MainUtils {
+
```

Comment:
ok , cool methods, but they should be static

Reply:
You're right, good catch. Made them static